index.coffee

```diff
@@ -18,6 +18,7 @@ module.exports = exports = (queueOpts = {}) ->
 		opts.retryStrategy ?= queueOpts.retryStrategy
 
 		requestRetry opts, (err, response, body) ->
+			done()
```

Comment:
This should probably be called with `done(err)` at a guess, so that the queue knows an error happened and that it should call the queue error handler instead of carrying on as normal